pyccel/ast/basic.py

```diff
@@ -1,6 +1,6 @@
 from sympy.core.basic import Basic as sp_Basic
 
-__all__ = ('Basic',)
+__all__ = ('Basic', 'PyccelAstNode')
```

Comment:
@saidctb: Now that we have a `PyccelAstNode` class, do we still need a separate `Basic` class? It is not clear to me why we need some Pyccel AST nodes to inherit from one class, and some from the other. Maybe it is just a matter of giving `Basic` a more meaningful name.

Reply:
it's not used yet, that's why we should keep it separate for the moment until we validate it
and set its attribute, we  then can merge it with Basic

Reply:
the attributes are not used that's what I mean
if you prefer we can wait until we validate the type inference mechanism with it

Reply:
For now we keep things as they are. We refer to issue #305 for improvements on our type inference mechanism.